cmd/tree-walk.go

```diff
@@ -147,7 +147,6 @@ func doTreeWalk(ctx context.Context, bucket, prefixDir, entryPrefixMatch, marker
 	if err != nil {
 		select {
 		case <-endWalkCh:
-			logger.LogIf(ctx, errWalkAbort)
```

Comment:
There is one more place we need to remove it, all the way below in the same file.